lms/static/js/verify_student/views/make_payment_step_view.js

```diff
@@ -98,7 +109,7 @@ var edx = edx || {};
 
         },
 
-        handleCreateOrderResponse: function( paymentParams ) {
+        handleCreateOrderResponse: function( paymentData ) {
             // At this point, the order has been created on the server,
```

Comment:
This is not longer always the case. If the ecommerce service is being used, a basket has been created and frozen on the server, but no order has been created.